web/src/lib/components/admin-page/settings/setting-accordion.svelte

```diff
@@ -1,10 +1,23 @@
 <script lang="ts">
+  import { page } from '$app/stores';
+  import { hasParamValue, updateParamList } from '$lib/utils';
   import { slide } from 'svelte/transition';
+
   export let title: string;
   export let subtitle = '';
-
+  export let key: string;
   export let isOpen = false;
-  const toggle = () => (isOpen = !isOpen);
+
+  const syncFromUrl = () => (isOpen = hasParamValue('isOpen', key));
+  const syncToUrl = (isOpen: boolean) => updateParamList({ param: 'isOpen', value: key, add: isOpen });
```

Comment:
In a previous PR, I put all query parameters in the `QueryParameter` enum in `constants.ts`, can you add it too ?